docs/manual/comparison_with_sql.rst

```diff
@@ -0,0 +1,662 @@
+
+.. code-block:: python
+
+    iris[:5, ('sepal_length', 'sepal_width', 'petal_length')]
+
+
+.. csv-table:: select columns
+    :header: "sepal_length", "sepal_width", "petal_length"
+    :widths: 10,10,10
+
+    5.1,	3.5,	1.4
+    4.9,	3,	1.4
+	4.7,	3.2,	1.3
+	4.6,	3.1,	1.5
+	5,	3.6,	1.4
+
+In ``SQL``, you can select all columns with the ``*`` symbol :
+
+.. code-block:: SQL
+
+    SELECT *
+    FROM iris
+    LIMIT 5;
+
+In ``datatable``, all columns can be selected with the :ref:`f-expressions` :
+
+.. code-block:: python
+
+    iris[:5, f[:]]
+
+.. csv-table:: select all columns
+    :header: "sepal_length", "sepal_width", "petal_length", "petal_width",	"species"
+    :widths: 10,10,10,10,10
+
+    5.1,	3.5,	1.4,	0.2,	setosa
+    	4.9,	3,	1.4,	0.2,	setosa
+    	4.7,	3.2,	1.3,	0.2,	setosa
+    	4.6,	3.1,	1.5,	0.2,	setosa
+    	5,	3.6,	1.4,	0.2,	setosa
+
+How about adding new columns? In ``SQL``, this is done also in the ``SELECT`` clause :
+
+.. code-block:: SQL
+
+    SELECT *,
+          sepal_length*2 as sepal_length_doubled
+    FROM iris
+    LIMIT 5;
+
+In ``datatable``, addition of new columns occurs in the ``j`` column :
+
+.. code-block:: python
+
+    iris[:5,
+         f[:].extend({"sepal_length_doubled": f.sepal_length * 2})]
+
+The :func:`update` option can also be used to add new columns. The operation occurs in-place; reassignment is not required.
+
+.. code-block:: python
+
+    iris[:, update(sepal_length_doubled = f.sepal_length * 2)]
+
+    iris[:5, :]
+
+.. csv-table:: Add a new column
+    :header:    sepal_length,	sepal_width,	petal_length,	petal_width,	species,	sepal_length_doubled
+    :widths: 10,10,10,10,10,10
+
+    	5.1,	3.5,1.4,	0.2,	setosa,	10.2
+	    4.9,	3,	1.4,	0.2,	setosa,	9.8
+	    4.7	,3.2,	1.3,	0.2,	setosa,	9.4
+	    4.6,	3.1,	1.5,	0.2,	setosa,	9.2
+	    5,	3.6,	1.4,	0.2,	setosa,	10
+
+
+FILTER
+-------
+
+Filtering in ``SQL`` is done via the ``WHERE`` clause.
+
+.. code-block:: SQL
+
+    SELECT *
+    FROM iris
+    WHERE species = 'viriginica'
+    LIMIT 5;
+
+In ``datatable``, filtration is done in the ``i`` section :
+
+.. code-block:: python
+
+    iris[f.species=="virginica", :].head(5)
+
+.. csv-table:: Filtration
+    :header:    sepal_length,	sepal_width,	petal_length,	petal_width,	species
+    :widths: 10,10,10,10,10
+
+    6.3,	3.3,	6,	2.5,	virginica
+	5.8,	2.7,	5.1,	1.9,	virginica
+	7.1,	3,	5.9,	2.1,	virginica
+	6.3,	2.9,	5.6,	1.8,	virginica
+	6.5,	3,	5.8,	2.2,	virginica
+
+Note that in ``SQL``, equality comparison is done with the ``=`` symbol, whereas in ``python``, it is with the ``==`` operator.
+You can filter with multple conditions :
+
+.. code-block:: SQL
+
+    SELECT *
+    FROM iris
+    WHERE species = 'setosa'
+    AND sepal_length = 5;
+
+In ``datatable`` each condition is wrapped in parentheses; the ``&`` operator is the equivalent of ``AND``, while ``|`` is the equivalent of ``OR``.
+
+.. code-block:: python
+
+    iris[(f.species=="setosa")&(f.sepal_length==5), :]
```

Comment:
For better readability I would add some spaces at least around `&`, something like this may also work
```
iris[(f.species == "setosa") & (f.sepal_length == 5), :]
```